### ISSUE TITLE
Implement public API for getting options from each validation.

### ISSUE THREAD

**Issue by phcoliveira** [offirgolan/ember-cp-validations#93]
HI there! Great work you've been doing. Thanks!

I haven't found a public API for getting the original options passed to each validation. I would like to use them in association with certain HTML attributes. Here is an example of what I would like to do:

``` handlebars
{{input value=model.email
        required=model.validations.attrs.email.validations.presence
        maxlength=model.validations.attrs.email.validations.length.max}}
```

I am using the ```v-get``` helper. I wrote the example that way trying to make myself clearer.

Anyway, I think that such API would be useful.

**Comment by offirgolan** [comment]
Hi @phcoliveira! Thanks for pointing this use case out to me. Let me take some time to think about the best way to support this :smile_cat:

**Comment by offirgolan** [comment]
Ill have a release out tonight

**Comment by offirgolan** [comment]
Tagged and released with `v2.2.0`
